<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SketcherSideBar</class>
 <widget class="QWidget" name="SketcherSideBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>95</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="schrodinger::sketcher::SelectOptionsWidget" name="select_options_wdg" native="true"/>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="schrodinger::sketcher::DrawToolsWidget" name="draw_tools_wdg" native="true"/>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="schrodinger::sketcher::RingToolWidget" name="ring_tool_wdg" native="true"/>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="schrodinger::sketcher::EnumerationToolWidget" name="enumeration_tool_wdg" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::SelectOptionsWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/select_options_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::DrawToolsWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/draw_tools_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::EnumerationToolWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/enumeration_tool_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>schrodinger::sketcher::RingToolWidget</class>
   <extends>QWidget</extends>
   <header>schrodinger/sketcher/ring_tool_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
</ui>
